src/platform/user/profile/reducers/index.js

```diff
@@ -61,7 +61,7 @@ function profileInformation(state = initialState, action) {
   switch (action.type) {
     case UPDATE_PROFILE_FIELDS: {
       const newState = mapRawUserDataToState(action.payload);
-      return merge(state, newState);
+      return Object.assign({}, state, newState);
```

Comment:
In a couple places I found `merge` uses that could be `assign` instead, because we are not actually looking to combine objects, but replace them.